From a0a7226c68cbb3150532d8dfcc8232d3d62e2bae Mon Sep 17 00:00:00 2001
From: =?utf-8?q?Caio=20Rom=C3=A3o?= <caioromao@gmail.com>
Date: Sun, 14 Jun 2009 21:42:37 -0300
Subject: [PATCH 05/10] [FIX] Indentation error on modify_attachment()

TODO: PEP8-ify the sources. At least converting the tabs to whitespace.
---
 bugz/bugzilla.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/bugz/bugzilla.py b/bugz/bugzilla.py
index ea7a143..6bd79a3 100644
--- a/bugz/bugzilla.py
+++ b/bugz/bugzilla.py
@@ -649,7 +649,7 @@ class Bugz:
 		except:
 			return []
 
-    def modify_attachment(self, attachid, **kwargs):
+	def modify_attachment(self, attachid, **kwargs):
 		"""Modifies an attachment given it's attachment_id
 
 		@param   attachid: attachment id
-- 
1.6.3.2

